### PR TITLE
QTL contains undefined functions

### LINKED ISSUE #1391: Bugfix: QTL report contains undefined functions.
primarily from `{dplyr}`

### PR DESCRIPTION
**Pull request by andersonjake1988**
## Overview
fix #1391 

## Test Notes/Sample Code

Notes:

### CODE REVIEW

File: gsm_QTL_report.html
Comment (mattroumaya) [inline gsm_QTL_report.html:1]: I've done this so many times, but make sure to delete this

**Review by mattroumaya — CHANGES_REQUESTED** [review]
I'm not able to run the QTL report with a local installation of this branch by running: 

```
library(gsm)

snapshot <- gsm::Make_Snapshot()

gsm::Study_Report(snapshot, strReportType = "QTL")
```
I'm getting the error:

```
Output created: gsm_country_report.html
> gsm::Study_Report(snapshot, strReportType = "QTL")


processing file: KRIReportByQTL.Rmd
  |...................................................                                      |  57% [unnamed-chunk-4]
Quitting from lines 54-65 [unnamed-chunk-4] (KRIReportByQTL.Rmd)
Error in `is.data.frame()`:
! object 'results_summary' not found
Backtrace:
 1. gsm::qtl_results(setup$active)
 5. dplyr:::left_join.data.frame(., results_summary)
 6. dplyr::auto_copy(x, y, copy = copy)
 8. dplyr:::same_src.data.frame(x, y)
 9. base::is.data.frame(y)
```

File: gsm_QTL_report.html
Comment (andersonjake1988) [reply gsm_QTL_report.html:1]: I knew it would happen one day, thanks for catching it!

**Review by mattroumaya — APPROVED** [review]
Excellent! 😎

**Pull request merged**